src/test/collaterals.sol

```diff
@@ -0,0 +1,1513 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity 0.6.12;
+
+contract Collaterals {
+    struct SystemValues {
+        uint256 line_offset;
+        uint256 pot_dsr;
+        uint256 pause_delay;
+        uint256 vow_wait;
+        uint256 vow_dump;
+        uint256 vow_sump;
+        uint256 vow_bump;
+        uint256 vow_hump_min;
+        uint256 vow_hump_max;
+        uint256 flap_beg;
+        uint256 flap_ttl;
+        uint256 flap_tau;
+        uint256 cat_box;
+        uint256 dog_Hole;
+        address pause_authority;
+        address osm_mom_authority;
+        address flipper_mom_authority;
+        address clipper_mom_authority;
+        uint256 ilk_count;
+        mapping (bytes32 => CollateralValues) collaterals;
+    }
+
+    struct CollateralValues {
+        bool aL_enabled;
+        uint256 aL_line;
+        uint256 aL_gap;
+        uint256 aL_ttl;
+        uint256 line;
+        uint256 dust;
+        uint256 pct;
+        uint256 mat;
+        bytes32 liqType;
+        bool    liqOn;
+        uint256 chop;
+        uint256 cat_dunk;
+        uint256 flip_beg;
+        uint48  flip_ttl;
+        uint48  flip_tau;
+        uint256 flipper_mom;
+        uint256 dog_hole;
+        uint256 clip_buf;
+        uint256 clip_tail;
+        uint256 clip_cusp;
+        uint256 clip_chip;
+        uint256 clip_tip;
+        uint256 clipper_mom;
+        uint256 cm_tolerance;
+        uint256 calc_tau;
+        uint256 calc_step;
+        uint256 calc_cut;
+        bool    offboarding; // if setting to true, we need to remove these  when we are done with the lerp
+    }
+
+    uint256 constant HUNDRED    = 10 ** 2;
+    uint256 constant THOUSAND   = 10 ** 3;
+    uint256 constant MILLION    = 10 ** 6;
+    uint256 constant BILLION    = 10 ** 9;
+    uint256 constant RAD        = 10 ** 45;
+
+    SystemValues afterSpell;
+
+    function setCollateralValues() public {
+        //
+        // Values for all collateral
```

Comment:
I would keep the previous comment or remark that the values would need to be updated accordingly with collateral changes (eg. Risk Parameter Adjustments) in spell actions:
`Test for all collateral based changes here`

Reply:
I guess my thinking was that this is not actually the test for collateral based changes, this is where the values are set so we can compare them later in the test.  How about adding a line: `Update when adding or modifying Collateral Values`

Reply:
To consider in #212

Reply:
added here: https://github.com/makerdao/spells-mainnet/pull/212/commits/cdef7c4bd1131591fd78b1282ef9c8bbc1eabefa